extra/resources/ifspeed

```diff
@@ -325,6 +340,24 @@ bridge_get_speed() {
     fi
 }
 
+hfi1_get_speed() {
+    local iface=$1
+    local hfi1_speed
+    local hfi1_speed
+    local hfi1_value
+    local hfi1_desc
+
+    # Currently (9/14/2017 Intel Omni Path v10.5.0.0.155) Intel doesn't have Dual/Multiple ports Host Channel Adapters
```

Comment:
actually this one is safe anyways.
$ read a b c < <(echo aa bb cc; echo dd ee ff) ; echo $a
aa